### PR TITLE
fix: stage doens't adapt replace notify for notification

### PR DESCRIPTION
**Pull request by 18202781743**
Notify server adapt replace notify, it will override old notify
both DB data and timeout data, and emit NotificationStateChanged.

pms: BUG-289863

### CODE REVIEW

File: panels/notification/center/notifystagingmodel.cpp [hunk 1]
```diff
@@ -237,6 +237,18 @@ NotifyEntity NotifyStagingModel::notifyById(qint64 id) const
     return {};
 }
 
+void NotifyStagingModel::replace(const NotifyEntity &entity)
+{
+    for (int i = 0; i < m_appNotifies.size(); i++) {
+        auto item = m_appNotifies[i];
+        if (item->entity().id() == entity.id()) {
```
Comment (yixinshark) [inline panels/notification/center/notifystagingmodel.cpp:244]: 这个地方满足了，下面是不是可以break.

**Review by yixinshark — DISMISSED** [review]

File: panels/notification/common/notifyentity.h [hunk 2]
```diff
@@ -76,6 +76,7 @@ class NotifyEntity
 
     uint replacesId() const;
     void setReplacesId(uint replacesId);
+    bool isReplace() const;
```
Comment (yixinshark) [inline panels/notification/common/notifyentity.h:79]: 这个是不是hasReplaceId()更合适

File: panels/notification/center/notifystagingmodel.cpp (hunk 1 above)
Comment (18202781743) [reply panels/notification/center/notifystagingmodel.cpp:244]: done,是可以break,

File: panels/notification/common/notifyentity.h (hunk 2 above)
Comment (18202781743) [reply panels/notification/common/notifyentity.h:79]: 是直接替换了，不是跟之前的共存，跟之前的通知共用同一个id, 应该是is吧,

**Review by yixinshark — APPROVED** [review]

**Pull request merged**